Kickstarter-iOS/Features/MessageBanner/Controller/MessageBannerViewController.swift

```diff
@@ -99,6 +108,7 @@ public final class MessageBannerViewController: UIViewController, NibLoading {
 
     if !isHidden {
       self.view.superview?.bringSubviewToFront(self.view)
+      self.view.superview?.isUserInteractionEnabled = false
```

Comment:
Did you want the message banner class to be responsible for turning on and off user interaction? I'm okay with that (as long as that works for all current call sites), but then the corresponding lines of code in the individual classes need to be deleted.

Reply:
Yep - It does work for the other call sites. Good catch. I thought I had gotten them all. Will remove it from the last 2 classes.